renderbus/rayvision_api
# feat: API structural optimization

- How to optimize the validation of interface parameters;
- Speed up requests with session optimization;
- Add the transfer json profile interface;
- Optimize the way some code is displayed and remove some useless functional code;
